app/wasm_filter.go

```diff
@@ -0,0 +1,34 @@
+package app
+
+import (
+	"strings"
+
+	"github.com/Finschia/finschia-sdk/codec"
+	codectypes "github.com/Finschia/finschia-sdk/codec/types"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	sdkerrors "github.com/Finschia/finschia-sdk/types/errors"
+	wasmkeeper "github.com/Finschia/wasmd/x/wasm/keeper"
+	wasmtypes "github.com/Finschia/wasmd/x/wasm/types"
+	wasmvmtypes "github.com/Finschia/wasmvm/types"
+)
+
+func filteredStargateMsgEncoders(cdc codec.Codec) *wasmkeeper.MessageEncoders {
+	return &wasmkeeper.MessageEncoders{
+		Stargate: wasmFilteredEncodeStargateMsg(cdc),
+	}
+}
+
+func wasmFilteredEncodeStargateMsg(unpakcer codectypes.AnyUnpacker) wasmkeeper.StargateEncoder {
+	// deny list in StargateMsg of wasm
+	deniedModulesInStargateMsg := []string{"/lbm.fswap.v1", "/lbm.fbridge.v1"}
```

Comment:
each "/lbm.fswap.v1" and "lbm.fbridge.v1" is not a module name, but a namespace of proto.

1. The variable name is a little confusing.
2. This makes confusing error messages later.

Reply:
> The variable name is a little confusing.
Good. If then, how about `deniedMsgInStargateMsg`?

> This makes confusing error messages later.
Could you suggest error message?

Reply:
A variable name should be a noun. "deniedMsgInStargateMsg" confuses that "Msg" is the main noun.
How about "deniedProtoNamespace" or "protoNamespacesDisallow"?

Reply:
I think it would be more appropriate to use "Msg" instead of "ProtoName".